sql/src/main/java/io/crate/analyze/expressions/ExpressionAnalyzer.java

```diff
@@ -91,11 +88,11 @@
 
     public ExpressionAnalyzer(AnalysisMetaData analysisMetaData,
                               ParameterContext parameterContext,
-                              Map<QualifiedName, AnalyzedRelation> sources) {
+                              FieldResolver sources) {
```

Comment:
please also rename it here and the internal attribute from sources to fieldresolver